src/python/grpcio_tests/commands.py

```diff
@@ -220,6 +220,7 @@ class TestGevent(setuptools.Command):
         'unit._cython._channel_test.ChannelTest.test_negative_deadline_connectivity',
         # TODO(https://github.com/grpc/grpc/issues/15411) enable this test
         'unit._local_credentials_test.LocalCredentialsTest',
+        'unit._alts_credentials_test.ALTSCredentialsTest',
```

Comment:
Does this test fail on gevent? Do you know why?

Reply:
I've mechanically included ALTS test here without actually validating if it fails or not - removed it from this list.